penny_university/middleware/integration_test_logging.py

```diff
@@ -0,0 +1,69 @@
+import logging
+from functools import wraps
+from unittest import mock
+
+from django.conf import settings
+from django.core.exceptions import MiddlewareNotUsed
+
+
+class IntegrationTestLogging:
```

Comment:
It would be cool to extend this to all requests and responses so we don't have to maintain mock data manually for frotend tests

Reply:
I don't completely understand your statement here. As stated, this _is_ available for all requests and responses, no?